pom.xml

```diff
@@ -213,6 +213,12 @@
       </exclusions>
     </dependency>
 
+    <dependency>
```

Comment:
Looking at the title of the PR, shouldn't `<groupId>com.google.guava</groupId>` be removed then? :-)

Reply:
you should read the comment here ;)  https://github.com/jenkinsci/jira-plugin/blob/master/pom.xml#L142